mergify_engine/context.py

```diff
@@ -1237,7 +1237,9 @@ async def items(self) -> typing.Dict[str, ContextAttributeType]:
     async def render_template(
         self,
         template: str,
-        extra_variables: typing.Optional[typing.Dict[str, str]] = None,
+        extra_variables: typing.Optional[
+            typing.Dict[str, typing.Union[str, bool]]
```

Comment:
This should probably be `typing.Any` since it's jus passing down objects, no?

Reply:
I explicitly put str, bool, to control what we pass to Jinja, I don't want to send list or dict or a real object.

Reply:
Well that could happen at some point for good reasons, no?

Reply:
It's Jinja, if we send a `context.Context()` in a template, I don't want to imagine what people can do with that.